src/tools/ceph_objectstore_tool.cc

```diff
@@ -1892,10 +1070,11 @@ int do_import(ObjectStore *store, OSDSuperblock& sb, bool force)
 
   //First section must be TYPE_PG_BEGIN
   sectiontype_t type;
-  ret = read_section(file_fd, &type, &ebl);
+  ret = read_section(&type, &ebl);
   if (ret)
     return ret;
   if (type != TYPE_PG_BEGIN) {
```

Comment:
It would be nice to see if (type == TYPE_POOL_BEGIN) and output that "A pool export can't be imported into a pg."  Instead of getting "Invalid first section type"   I'll add it later if it doesn't make it into this branch.